### PR TITLE
Refactor `ModelExperimental` constructor options

### PR DESCRIPTION
**Pull request by ptrgags**
The constructors `ModelExperimental.fromGltf()`, `.fromB3dm()`, `fromI3dm()`, and `fromPnts`() have a large number of options, and we've been forgetting to update them consistently. 

This PR factors out the initialization of the options object into a single private function, `makeModelOptions()`, so there's only one place to update the options.

@j9liu could you review this when you get a chance?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/refactor-model-experimental-options/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Source/Scene/ModelExperimental/ModelExperimental.js [hunk 1]
```diff
@@ -1233,32 +1233,11 @@ ModelExperimental.fromGltf = function (options) {
...
-    featureIdLabel: options.featureIdLabel,
-    instanceFeatureIdLabel: options.instanceFeatureIdLabel,
-    pointCloudShading: options.pointCloudShading,
-    lightColor: options.lightColor,
-    backFaceCulling: options.backFaceCulling,
-    shadows: options.shadows,
-    showCreditsOnScreen: options.showCreditsOnScreen,
-  };
+  const modelOptions = makeModelOptions(loader, type, options);
+  // Due to the logic above for determining the resources, use the computed
```
Comment (j9liu) [inline Source/Scene/ModelExperimental/ModelExperimental.js:1237]: Is there a reason for this comment? (i.e. is this something important that wasn't previously documented with the old model options object?)

**Review by j9liu — COMMENTED** [review]
@ptrgags this looks good! I just had one question about a comment that got added, but besides that once I can merge once CI passes.

File: Source/Scene/ModelExperimental/ModelExperimental.js (hunk 1 above)
Comment (ptrgags) [reply Source/Scene/ModelExperimental/ModelExperimental.js]: it was to point out that the variable name is slightly different (`resource` vs `gltfResource`). But eh that's clear enough from this line of code. I removed the comment.

**Comment by ptrgags** [comment]
@j9liu updated!

**Comment by j9liu** [comment]
I'm running into a local failure in `Model` that seems unrelated to the changes in this PR. I'll write up an issue with more details, but I don't think it should hold up merging this into `main`. Thanks @ptrgags !

**Pull request merged**